### PR TITLE
7140 vre contact info

### PR DESCRIPTION
**Pull request by rianfowler**
Need to reconcile the phone number labels:
design: day, evening
schema: home, mobile

For now I used the design labels. 
screen shot 2018-02-19 at 10 56 25
screen shot 2018-02-19 at 10 56 17

### CODE REVIEW

File: src/js/vre/chapter36/config/form.js [hunk 1]
```diff
@@ -6,19 +6,24 @@ import {
   genderLabels
 } from '../../../common/utils/labels.jsx';
 
+import * as address from '../../../common/schemaform/definitions/address';
```
Comment (rianfowler) [inline src/js/vre/chapter36/config/form.js]: Make sure this is the correct schema to be used.

**Comment by rianfowler** [comment]
Figure we can go ahead and merge these and [update](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/7140#issuecomment-366721762) the field labels / schema props for the phone numbers in another PR.

File: src/js/vre/chapter36/config/form.js (hunk 1 above)
Comment (jbalboni) [reply src/js/vre/chapter36/config/form.js]: It is, as far as I know.

File: test/vre/chapter36/config/contactInformation.unit.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+      fillData(form, 'select#root_applicantAddress_state', 'CA');
+      fillData(form, 'input#root_applicantAddress_postalCode', '90034');
+
+      form.find('form').simulate('submit');
+
+      expect(form.find('.usa-input-error').length).to.equal(0);
+      expect(onSubmit.called).to.be.true;
+    });
+  });
+  describe('contact information', () => {
```
Comment (jbalboni) [inline test/vre/chapter36/config/contactInformation.unit.spec.jsx]: We normally put each page in a separate file.

**Review by jbalboni — APPROVED** [review]
LGTM.

**Pull request merged**